### DISCUSSION TITLE
System.Drawing.Colors or System.Drawing.Color?

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1510] [General] [answered]
:wave: 

Looking at generated MagickColors, it refers to class `System.Drawing.Colors`.
However, I find only [System.Drawing.Color](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.color?view=net-8.0).

Wrong comment/reference?

Thanks.

**Comment by dlemstra** [comment, marked as answer]
Yeah that does look like a typo. But please don't open a pull request. I am working on changing this to use a source generator instead.
